resources/docker/images/authoring/tomcat/Dockerfile

```diff
@@ -0,0 +1,165 @@
+# Copyright (C) 2007-2023 Crafter Software Corporation. All Rights Reserved.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as published by
+# the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+FROM eclipse-temurin:17-jdk-alpine AS main
+
+############# INSTALL NATIVE TOMCAT #############
+ENV CATALINA_HOME=/usr/local/tomcat \
+    TOMCAT_MAJOR="9" \
```

Comment:
The Dockerfile seems incorrect for `develop` branch where we should use Java 21 and Tomcat 10. We have the latest on https://github.com/craftercms/docker-images/tree/develop